### PR TITLE
Fix dependabot

### PR DESCRIPTION
**Pull request by djbender**
this repo was missing the dependabot specific configuration for automated workflows.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+    types: [labeled]
+  push:
+    branches-ignore:
+      - "dependabot/**"
+  repository_dispatch:
 
 jobs:
   main:
     runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'dependencies') || github.event_name == 'push'
```
Comment (rexfleischer) [inline .github/workflows/ci.yml:14]: Will this make it not run if it's a repository_dispatch event?

Comment (djbender) [reply .github/workflows/ci.yml:14]: looks like it! i'll update this.

**Review by rexfleischer — APPROVED** [review]
This change looks good to me.

I'm a little confused though, because (if I'm reading this correctly) this change will cause dependabot updates to go untested. Is that intended?

**Comment by djbender** [comment]
@rexfleischer the first part of the conditional causes dependabot PRs to get tested. They're always labeled with 'dependencies.'

**Comment by rexfleischer** [comment]
> @rexfleischer the first part of the conditional causes dependabot PRs to get tested. They're always labeled with 'dependencies.'

Yes, but the `dependabot/**` ignore branches?

**Comment by djbender** [comment]
ah yes. it's the pull_request_target that handles it :)

**Pull request merged**